static/docs/commands-reference/version.md

```diff
@@ -0,0 +1,41 @@
+# version
+
+This command shows the system/environment information along with the DVC version.
+
+## Synopsis
+
+```usage
+    usage: dvc version [-h] [-q | -v]
+```
+
+## Description
+
+Running the command `dvc version` outputs the
```

Comment:
please, let's move some stuff to the previous line. It looks weird now a little bit.